### PR TITLE
Add Llama2-MedTuned-Instructions dataset

### PR DESCRIPTION
**Pull request by manandey**
All tests are passing. c.c @shayne-longpre

### CODE REVIEW

File: data_summaries/Llama2-MedTuned-Instructions.json
Comment (wwbrannon) [inline data_summaries/Llama2-MedTuned-Instructions.json:44]: left over from the template?

File: data_summaries/Llama2-MedTuned-Instructions.json
Comment (wwbrannon) [inline data_summaries/Llama2-MedTuned-Instructions.json:54]: the HF page says "cc-by-nc-4.0" -- if that doesn't apply I think the note should explain why?

**Review by wwbrannon — CHANGES_REQUESTED** [review]

File: data_summaries/Llama2-MedTuned-Instructions.json
Comment (manandey) [reply data_summaries/Llama2-MedTuned-Instructions.json]: Sorry, I had mentioned "No License" earlier since there was no license mentioned when I raised the PR. The author updated the license after I created an issue here (https://huggingface.co/datasets/nlpie/Llama2-MedTuned-Instructions/discussions/2) -- updated the PR now,

File: data_summaries/Llama2-MedTuned-Instructions.json
Comment (shayne-longpre) [inline data_summaries/Llama2-MedTuned-Instructions.json:18]: There are so biomedical QA task categories too in the constants/. Are those included in this dataset?

File: data_summaries/Llama2-MedTuned-Instructions.json
Comment (shayne-longpre) [inline data_summaries/Llama2-MedTuned-Instructions.json:53]: Could you link this instead since it's a bit more precise? https://huggingface.co/datasets/nlpie/Llama2-MedTuned-Instructions/discussions/2

**Review by shayne-longpre — APPROVED** [review]
LGTM after these comments and Will approves

**Comment by manandey** [comment]
Hi @wwbrannon, I have addressed all the comments. Feel free to merge the PR according to your convenience. Thanks!

**Review by wwbrannon — APPROVED** [review]

**Pull request merged**
